test/integration/default/serverspec/test_server_running_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'serverspec'
+set :backend, :exec
+
+# timesync app
+describe port(8000) do
+  it { should be_listening }
+end
+
+# haproxy
+describe port(80) do
+  it { should be_listening }
+end
+
+# database exists
+describe file('/opt/timesync/source/dev.sqlite3') do
```

Comment:
Since we are chowning this file, you should actually check the ownership on it for now.